style.css

```diff
@@ -443,3 +457,136 @@ button:hover {
   top: 20px;
   margin: auto;
 }
+
+/* Desktop version */
+
+@media (min-width: 768px) {
+  nav img {
+    display: none;
+  }
+
+  .header {
+    display: flex;
+  }
+
+  .container {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-content: center;
+    flex-wrap: wrap;
+  }
+
+  .container h1 {
+    align-self: flex-start;
+  }
+
+  .card {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    padding: 24px 22px 24px 24px;
+  }
+
+  .card2 img {
+    max-width: 544px;
+    height: 448px;
+    border-radius: 8px;
+    order: 1;
+  }
+
+  .card1 img {
+    max-width: 544px;
+    border-radius: 8px;
+    height: 448px;
+    order: -1;
+  }
+
+  .card--content {
+    height: 465px;
+    max-width: 544px;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    align-content: center;
+    flex-wrap: wrap;
+    flex-grow: 1;
+    gap: 30px;
+    padding: 24px;
+  }
+
+  .card--content .title {
+    align-self: flex-start;
+    font-size: 40px;
+  }
+
+  #AboutmySelf {
```

Comment:
The color for the `About` section title should be `#091E42` and the color for the `Get My Resume` button text should be `#396DF2`. The `Get My Resume` button should have a hover effect as defined in the design. Moreover, `Languages` should have icons as can be seen in the design.


Your design:
![your-languages](https://user-images.githubusercontent.com/35504149/172493313-7861581f-60a1-4fa3-9187-2ddf1595aca7.png)


Original design:
![languages](https://user-images.githubusercontent.com/35504149/172493339-1ffd36af-31eb-4661-9095-37f486bbd668.png)